uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-notification.component.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-env amd */
+
+define([
+  'leave-absences/shared/modules/components'
```

Comment:
this applies not only to this particular line but in general, why anything related to the notification badge should be shared?